AUTHORS.txt

```diff
@@ -183,3 +183,4 @@ YetheSamartaka
 xrufix
 Zakant <Zakant@gmx.de>
 zGuba
+QuantX
```

Comment:
Place it up above so it is in alphabetical order

Reply:
Done. I somehow missed that this was in alphabetical order.